setup.py

```diff
@@ -1,38 +1,38 @@
 from setuptools import setup
 
 dependencies = [
-    "multidict==5.1.0",  # Avoid 5.2.0 due to Avast
-    "aiofiles==0.7.0",  # Async IO for files
-    "blspy==1.0.8",  # Signature library
-    "chiavdf==1.0.3",  # timelord and vdf verification
-    "chiabip158==1.0",  # bip158-style wallet filters
-    "chiapos==1.0.7",  # proof of space
+    "multidict==5.2.0",  # Avoid 5.2.0 due to Avast
```

Comment:
Note that we need to avoid 5.2.0, as the comment says, to not get an anti virus false positive.

Reply:
Oops, I thought I removed that before pushing